pkg/asset/machines/master.go

```diff
@@ -29,7 +29,31 @@ type Master struct {
 	UserDataSecretRaw []byte
 }
 
-var _ asset.Asset = (*Master)(nil)
+var (
+	_ asset.Asset = (*Master)(nil)
+
+	defaultMachineClass = map[string]string{
```

Comment:
aws specific code should live in `pkg/asset/machines/aws`

Reply:
also WDYT using m4 by default and overriding only some regions, so that we only maintain override list?

Reply:
your hope was that I only keep non-m4 regions in this map instead of every region?   I can probably do that, and fix CI.